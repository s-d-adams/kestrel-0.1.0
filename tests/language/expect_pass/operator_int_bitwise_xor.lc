(inc"stdio.h")(def main(imp int)(do(printf"%d\n%d\n%d\n%d\n"(^ 21 42)(^ 21 63)(^ 63 21)(^ 0 0))0))