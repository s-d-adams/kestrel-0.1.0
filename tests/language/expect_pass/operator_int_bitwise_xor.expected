63
42
42
0
